lib/python/picongpu/plugins/plot_mpl/base_visualizer.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib.pyplot as plt
+
+
+class Visualizer(object):
+    def __init__(self, run_directory):
+
```

Comment:
`run_directory` parameter: please add [the usual docstring](https://github.com/ComputationalRadiationPhysics/picongpu/blob/2c8a2bdd74ccf56c3db78c62d4d33c58110b3533/lib/python/picongpu/plugins/phase_space.py#L62-L68)